src/transformers/models/imagebind/processing_imagebind.py

```diff
@@ -15,18 +15,21 @@
 Image/Text processor class for ImageBind
 """
 
+from typing import List, Union
+
+try:
+    from typing import Unpack
+except ImportError:
+    from typing_extensions import Unpack
+
+from ...image_utils import ImageInput
 from ...processing_utils import ProcessingKwargs, ProcessorMixin
-from ...tokenization_utils_base import BatchEncoding
+from ...tokenization_utils_base import BatchEncoding, PreTokenizedInput, TextInput
 
 
 class ImageBindProcessorKwargs(ProcessingKwargs, total=False):
```

Comment:
I believe we need to create custom `AudioKwargs` and `ImageKwargs` as we have arguments that are not in the default classes

Reply:
Yeah, need to discuss the custom kwargs so left them out. i will push those changes